ytfzf

```diff
@@ -94,11 +98,18 @@ function_exists "downloader" || downloader () {
 : ${is_auto_select=0}
 : ${is_random_select=0}
 
+# option parsing
+: ${long_opt_char:=-}
 
 # scrape
 : ${scrape=youtube}
 : ${sub_link_count=10}
 
+# Option Checks {{{
```

Comment:
what about a case statement here?
like checking for a `-` or `v`, if anything else then we check for non alphanumberic

Reply:
we could also add
```sh
[!v-]) print_info "warning: only v and - are reserved for longopts, other chars may be used in the future" ;;
```

to the case statement if we want to use other nonalphanumeric chars but i dont think we should.

Reply:
Yeah sounds good